pytorch_blade/benchmark/TorchBench/test_torch_bench.sh

```diff
@@ -12,23 +12,24 @@
 # !/bin/bash
 # install dependencies
 python3 -m virtualenv venv --system-site-packages && source venv/bin/activate
-pip3 install -U pip
-pip3 install -q librosa torchvision torchaudio torchtext pycocotools==2.0.3 --extra-index-url https://download.pytorch.org/whl/cu113
-
 script_dir=$(cd $(dirname "$0"); pwd)
+python3 -m pip install -q -r $script_dir/requirements.txt
+
 # setup for torchbenchmark
-OLD_HOME=$HOME
-benchmark_repo_dir=$OLD_HOME/.cache/torchbenchmark
-if [ ! -d $benchmark_repo_dir ]; then
-    git clone -q https://github.com/pytorch/benchmark.git --recursive $benchmark_repo_dir
+benchmark_repo_dir=$HOME/.cache/torchbenchmark
+if [ -d $benchmark_repo_dir ]; then
+    rm -rf $benchmark_repo_dir
 fi
-cd $benchmark_repo_dir && export HOME=$(pwd) && git lfs install --force && git pull && git submodule update --init --recursive --depth 1 && python3 install.py --continue_on_fail
+git clone -q https://github.com/pai-disc/torchbenchmark.git --recursive $benchmark_repo_dir
+# CI git-lfs permission problems
+cd $benchmark_repo_dir && export HOME=$(pwd) && git lfs install --force
+git pull && git submodule update --init --recursive --depth 1 && python3 install.py --continue_on_fail
 pushd $script_dir # pytorch_blade/benchmark/TorchBench
 # setup for torchdynamo
-ln -s $benchmark_repo_dir torchbenchmark 
-git clone -q https://github.com/pytorch/torchdynamo.git dynamo && pip3 install -q dynamo/
+ln -s $benchmark_repo_dir torchbenchmark
 
-# dynamo frontend and disc backend
-python3 blade_bench.py --backend blade_disc_compiler -d cuda --isolate --float32 --skip-accuracy-check 2>&1 | tee speedup_blade_disc_compiler.log
-cat speedup_blade_disc_compiler.csv
-popd
\ No newline at end of file
+# torchscript/torchdynamo frontend and disc backend
+TORCHBENCH_TOL=1e-2 python3 torchbenchmark/.github/scripts/run-config.py -c blade_bench.yaml -b ./torchbenchmark/ --output-dir .
```

Comment:
where is blade_bench.yaml?

Reply:
It's in the [torchbenchmark repo](https://github.com/pai-disc/torchbenchmark/blob/main/configs/blade_bench.yaml), because the script find config files in this folder.